spotbugsTestCases/src/java/ghIssues/Issue2436.java

```diff
@@ -0,0 +1,10 @@
+package ghIssues;
+
+import java.io.Serializable;
+
+public class Issue2436 implements Serializable {
+    public void doNothing(){
```

Comment:
Is this function really needed for a minimal example?

Reply:
true, removed it